torch/nn/quantized/modules/conv.py

```diff
@@ -498,3 +502,64 @@ def from_float(cls, mod):
         qconv.zero_point = int(act_zp)
 
         return qconv
+
+
+# === Transposed Convolutions ===
+
+class _ConvTransposeNd(_ConvNd):
+    def __init__(self, in_channels, out_channels, kernel_size, stride,
+                 padding, dilation, transposed, output_padding,
```

Comment:
also I think we probably don't need transposed, it should always be true for ConvTranspose right?

Reply:
This is forr the consistency with the FP codebase

Reply:
this doesn't make sense, shall we change it?

Reply:
can be a different PR